### PR TITLE
chrome for testing feature

### PR DESCRIPTION
**Pull request by udarrr**

### CODE REVIEW

**Comment by udarrr** [comment]
@christian-bromann Could you check it ?

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "selenium-standalone",
-  "version": "8.3.0",
+  "version": "9.0.0",
```
Comment (christian-bromann) [inline package.json:3]: We should revert this as this project has its own pipeline for releasing and bumping the version.

**Review by christian-bromann — COMMENTED** [review]
One comment, but otherwise looks good to me 👍 
great job!

File: package.json (hunk 1 above)
Comment (udarrr) [reply package.json:3]: to the 8.3.0 ?

Comment (christian-bromann) [reply package.json:3]: yes

Comment (udarrr) [reply package.json]: done :)

**Review by christian-bromann — APPROVED** [review]
Fantastic work 👏 
Thank you!

**Review by tehhowch — COMMENTED** [review]
Did adding support for "Chrome for Testing" _require_ also modifying the node version requirements? I think the only "reason" is because a totally new dependency was added (on the [`@puppeteers/browsers`](https://www.npmjs.com/package/@puppeteer/browsers) package). A backport to v8.x that is still compatible with node 12 would be useful to lots of folks that cannot yet use Webdriver v8 / Node 16 (e.g. because sync to async migrations are still ongoing, or other internal reasons).

**Pull request merged**